dev/how-to/deploy/tidb-binlog.md

```diff
@@ -358,6 +386,8 @@ The following part shows how to use Pump and Drainer based on the nodes above.
             the address through which Drainer provides the service (-addr="192.168.0.13:8249")
         -c int
             the number of the concurrency of the downstream for replication. The bigger the value, the better throughput performance of the concurrency (1 by default).
+        -cache-binlog-count int
+            the limit on the number of binlogs in the cache (65536 by default)
```

Comment:
```suggestion
            the limit on the number of binlog files in the cache (65536 by default)
```

Reply:
I think the description 'files' is not suitable. In this part 'binlogs' are struct instances but not files. Maybe the description 'items' is better.

Reply:
@lichunzhu Got it. Thanks for the explanation! 
@TomShawn Then let's use "items" here.

Reply:
OK.